lib/property_widgets/border_radius_property.dart

```diff
@@ -109,7 +110,9 @@ class _BorderRadiusPropertyState extends State<BorderRadiusProperty> {
             onPressed: () async {
               await show();
               widget.property.value = _currentBorderRadius;
-              widget.onChanged(widget.property);
+              if (widget.onChanged != null) {
```

Comment:
we can use Elvis operator here instead right? `widget.onChange?.call(widget.property)`